bff/src/Bff/DynamicFrontends/Internal/FrontendCollection.cs

```diff
@@ -14,10 +14,10 @@ internal class FrontendCollection : IDisposable, IFrontendCollection
     private readonly object _syncRoot = new();
 
     /// <summary>
-    /// Backing store for the frontends. This is marked 'volatile' because it can be read / updated from multiple threads.
```

Comment:
Discussed with @josephdecock and volatile is not needed here. Interlocked.Exchange handles all read optimization issues.